run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+declare -a EXTRA_FLAGS=()
+
+# Display Socket
+if [[ $XDG_SESSION_TYPE == "wayland" ]]
+then
+    WAYLAND_SOCKET=${WAYLAND_DISPLAY:-"wayland-0"}
+
+    if [[ -e "$XDG_RUNTIME_DIR/${WAYLAND_SOCKET}" ]]
+    then
+        echo "Wayland socket is available, running natively on Wayland."
+        echo "To disable, remove the --socket=wayland permission."
+        EXTRA_FLAGS+=(
+            "--enable-features=UseOzonePlatform,WaylandWindowDecorations"            
+            "--ozone-platform=wayland"
+            "--ozone-platform-hint=auto"            
```

Comment:
You don't need both these flags. The last one is the latest one, the previous one is deprecated. You should check what flag your Electron version supports.